packages/yoga/src/Avatar/native/Avatar.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { BuildingFilled } from '@gympass/yoga-icons';
+import { string, func, number } from 'prop-types';
+import Box from '../../Box';
+import Image from './Image';
+import Placeholder from './Placeholder';
+
+const Avatar = ({
+  src,
+  fill,
+  placeholder,
+  width = 48,
+  height = 48,
+  borderRadius = 'small',
+  type = 'default',
+}) => {
+  const icon = placeholder || BuildingFilled;
+  return (
+    <Box
+      bgColor="elements.selectionAndIcons"
+      display="flex"
+      width={width}
+      height={height}
+      alignItems="center"
+      justifyContent="center"
+      elevation="small"
+      borderRadius={borderRadius}
```

Comment:
We could spread the other properties here, this way it would make it easier to customize the component and add other properties. The same goes for `Circle`.

```js
borderRadius={borderRadius}
{...otherProps}
```

Reply:
@leonardokl Done!